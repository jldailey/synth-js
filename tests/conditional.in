a
	!? lt IE 9
		b
	c
