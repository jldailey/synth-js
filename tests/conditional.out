<a>
  <!--[if lt IE 9]>
    <b/>
  <![endif]-->
  <c/>
</a>
